content/organization/events/speakers/index.en.md

```diff
@@ -27,6 +27,48 @@ weight: 53
 
 * Remember the [rules about Prioritization of Underrepresented/Minority Genders](/about/mission/#r-ladies-rules--guidelines).
 
+* In line with R-Ladies’ Mission Statement and with our [Black Lives Matter Statement](https://blog.rladies.org/post/2020-06-06-blm/) 
+to achieve participation and representation in our events of generally excluded groups, taking into account the intersectionality of gender, race, language and geography; for these reasons we request:
+
+  - In chapter events, invite speakers of diverse race, language and geography (i.e., avoid a one year period with all white speakers at the chapter events).
+  - For panel discussions of 3 or more panelists, invite speakers of diverse race, language and geography (i.e., avoid all white panels).
+We need to do for the minority groups inside our community the same that we request other communities and spaces do for women and gender minorities.  
+### How to find speakers? 
+- You can use the [R-Ladies Directory](https://rladies.org/directory/) to identify potential speakers that are not: white, native anglophone, or from high income level countries.
+- You can reach out to other communities to request for speakers information (see lists below).
```

Comment:
```suggestion
- You can check similar directories and reach out to other communities (see lists below).
```

Reply:
This was suggested by Hannah but I committed the change by Maëlle instead of this one that was the one we want to.